db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -1585,6 +1582,14 @@ class NonBatchedOpsStressTest : public StressTest {
       } else {
         s = sst_file_writer.Put(k, v);
       }
+
+      if (s.ok()) {
```

Comment:
It seem we do rollback below if s is not ok?

Reply:
I see, I missed the ok check in the for loop.